server/api_server/api_server_start.py

```
@@ -6,6 +6,7 @@
 
 from flask import Flask, redirect, request, jsonify, url_for, Response
 from models.device_instance import DeviceInstance  # noqa: E402
+from models.parameters_instance import ParametersInstance  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)